releases/nightly-build/compose-files/docker-compose-nexus-redis-no-secty.yml

```diff
@@ -218,21 +218,29 @@ services:
       Service_Host: edgex-app-service-configurable-rules
       MessageBus_SubscribeHost_Host: edgex-core-data
       edgex_registry: consul://edgex-core-consul:8500
+      Binding_PublishTopic: events
     depends_on:
       - consul
       - logging
       - data
 
   rulesengine:
-    image: nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
+    image: emqx/kuiper:0.3.0
     ports:
-      - "48075:48075"
-    container_name: edgex-support-rulesengine
-    hostname: edgex-support-rulesengine
+      - "9081:9081"
+      - "20498:20498"
+    container_name: edgex-kuiper
+    hostname: edgex-kuiper
     networks:
       edgex-network:
         aliases:
-          - edgex-support-rulesengine
+          - edgex-kuiper
+    environment:
+      EDGEX_SERVER: edgex-app-service-configurable-rules
+      EDGEX_SERVICE_SERVER: http://edgex-core-data:48080
```

Comment:
Why do you need this?

Reply:
- ``edgex-app-service-configurable-rules`` is used for listening data from application service.
- ``http://edgex-core-data:48080`` is used for getting data types of data that is flowed into the Kuiper, and Kuiper will try to convert the ``EdgeX data type`` to ``Kuiper data type``. Refer to this link for the [data type conversions](https://github.com/emqx/kuiper/blob/master/docs/en_US/rules/sources/edgex.md).